utils/build-script-utils/src/version.rs

```diff
@@ -22,7 +22,7 @@ pub fn generate_cargo_keys() {
 	let commit = if let Ok(hash) = std::env::var("SUBSTRATE_CLI_GIT_COMMIT_HASH") {
 		Cow::from(hash.trim().to_owned())
 	} else {
-		match Command::new("git").args(&["rev-parse", "--short", "HEAD"]).output() {
+		match Command::new("git").args(&["rev-parse", "--short=11", "HEAD"]).output() {
```

Comment:
Maybe some comment that this `11` is important :D

Reply:
Good idea!